uint8   robot_id
uint8   topic_id
time    ts_limit
---
uint8[] headers
